lib/slack_announcer.rb

```diff
@@ -7,22 +7,33 @@ def initialize(environment_name, slack_url)
   end
 
   def announce_start(repo_name, application, slack_channel = "#govuk-deploy")
+    return unless staging_or_production?
```

Comment:
Do we know why we need this, because at the moment we don't see these messages about integration?

Reply:
I've just refactored this check slightly - previously it was in a subsequent method call, I've moved the return to the earliest opportunity.

Reply:
Ooh yes, okay, I missed that in the diff. Makes sense 👍🏻